app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,170 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$light_gray: #F2F2F2;
+$white: #FFF;
+$beach-blue: #5bc0de;
+
+body {
+  padding-top: 20px;
+}
+
+#index-background-header {
+  text-align: center;
+  padding-top: 300px;
+  font-family: Circular,"Helvetica Neue",Helvetica,Arial,sans-serif;
+  color: #FFF;
+  font-size: 5em;
+  font-weight: bold;
+}
+
+#search-bar-container .well{
+  background: #000;
+  background: rgba(0, 0, 0, 0.1);
+  position: absolute;
```

Comment:
Properties should be ordered background, background, border-color, bottom, left, margin-bottom, position, right